From 62454eab40ac79564f875a6d0770c65c67b68881 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 4 Jul 2018 11:24:35 +0800
Subject: [PATCH 4236/5242] MLK-18724-2 clk: imx8mq: remove
 IMX8MQ_CLK_NAND_USDHC_BUS_CG out from
 clks_init_on[]

commit  65774292462f65d4fed825caf005e05b768fd8fd from
https://source.codeaurora.org/external/imx/linux-imx.git

No need to enable IMX8MQ_CLK_NAND_USDHC_BUS_CG during the imx8mq clock driver
init, so remove it from the clks_init_on[].

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
(cherry picked from commit 6c90e1bfc38eab27921d26b1218993e5cd52a425)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mq.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/imx/clk-imx8mq.c b/drivers/clk/imx/clk-imx8mq.c
index 80adc75..5eca458 100644
--- a/drivers/clk/imx/clk-imx8mq.c
+++ b/drivers/clk/imx/clk-imx8mq.c
@@ -280,7 +280,7 @@
 static int const clks_init_on[] __initconst = {
 	IMX8MQ_CLK_DRAM_CORE, IMX8MQ_CLK_AHB_CG,
 	IMX8MQ_CLK_NOC_CG, IMX8MQ_CLK_NOC_APB_CG,
-	IMX8MQ_CLK_USB_BUS_CG, IMX8MQ_CLK_NAND_USDHC_BUS_CG,
+	IMX8MQ_CLK_USB_BUS_CG,
 	IMX8MQ_CLK_MAIN_AXI_CG, IMX8MQ_CLK_A53_CG,
 	IMX8MQ_CLK_TMU_ROOT,
 	IMX8MQ_CLK_DRAM_APB_DIV,
-- 
1.7.9.5

